docs/atlascli/command/atlas-clusters-update.txt

```diff
@@ -12,7 +12,13 @@ atlas clusters update
    :depth: 1
    :class: singlecol
 
-Update a MongoDB Atlas cluster.
+Modify the settings for your MongoDB Atlas cluster.
```

Comment:
I would use "Change" instead of "Modify"

Reply:
Modify is used a few places in our style guide as a recommended example, so I think either work in this context.